### PR TITLE
Loki: Add autocomplete hints for label_format and line_format

### PR DESCRIPTION
**Pull request by caleb15**
**What this PR does / why we need it**:
Adds documentation to label_format and line_format
Why it's needed: Current documentation is very minimal.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by ivanahuckova — APPROVED** [review]

**Comment by ivanahuckova** [comment]
We can't merge it because the ci build is failing. This is due to the linting. Can you please run `yarn run prettier:write` which will fix linting errors.

File: public/app/plugins/datasource/loki/syntax.ts [hunk 1]
```diff
@@ -79,12 +79,12 @@ export const PIPE_OPERATORS: CompletionItem[] = [
   {
     label: 'label_format',
     insertText: 'label_format',
-    documentation: 'Only available in Loki 2.0+.',
+    documentation: 'Use to rename, modify or add labels. For example, | label_format foo=bar . Only available in Loki 2.0+.',
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/loki/syntax.ts:82]:
```suggestion
    documentation: 'Use to rename, modify or add labels. For example, | label_format foo=bar. Only available in Loki 2.0+.',
```

File: public/app/plugins/datasource/loki/syntax.ts [hunk 2]
```diff
@@ -79,12 +79,12 @@ export const PIPE_OPERATORS: CompletionItem[] = [
...
   },
   {
     label: 'line_format',
     insertText: 'line_format',
-    documentation: 'Only available in Loki 2.0+.',
+    documentation: 'Rewrites log line content. For example, | line_format "{{.query}} {{.duration}}" . Only available in Loki 2.0+.',
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/loki/syntax.ts:87]:
```suggestion
    documentation: 'Rewrites log line content. For example, | line_format "{{.query}} {{.duration}}". Only available in Loki 2.0+.',
```

File: public/app/plugins/datasource/loki/syntax.ts (hunk 1 above)
Comment (caleb15) [reply public/app/plugins/datasource/loki/syntax.ts:82]: This was actually on purpose so people don't accidentally copy the period

File: public/app/plugins/datasource/loki/syntax.ts (hunk 2 above)
Comment (caleb15) [reply public/app/plugins/datasource/loki/syntax.ts:87]: This was actually on purpose so people don't accidentally copy the period

**Comment by caleb15** [comment]
@ivanahuckova do you happen to know what's wrong with the linting off the top of your head, so I can quickly fix it through the github website? I don't have the necessary node version to run prettier. If not I can install nvm and get the right node version.

**Comment by ivanahuckova** [comment]
Space after `documentation:` needs to be removed image

**Comment by ivanahuckova** [comment]
Awesome! Thanks @caleb15 🙏 .

**Pull request merged**